### ISSUE TITLE
Continue swagpath from where you left off

### ISSUE THREAD

**Issue by limikael** [tunapanda/wp-swag#26]
If you go through half the swagpath and then close your browser and then re-open it, it should be possible to resume from where you left off.

**Comment by philosofree** [comment]
Yes, and when one swag on the path is completed, the next in the swagpath should then be opened up.
This requires looking up what is just completed in prerequisites, and ensuring that each of those is then available in the swagmap 
eg https://contemplativefire.net/lisn/swag/map/ - some of these have been completed but only the original unlinked paths are showing.
